src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stopped working");
+

```

Comment:
use override if possible
```suggestion

    @Override
    public void stopWork() {
        System.out.println("Truck stopped working");
```